internal/backend/workloadproxy/reconciler.go

```diff
@@ -94,60 +110,50 @@ func (registry *Reconciler) Reconcile(cluster resource.ID, aliasToUpstreamAddres
 }
 
 // ensureLB ensures that a load balancer exists and started for the given cluster and alias, targeting the given upstream addresses.
-func (registry *Reconciler) ensureLB(lbSts *lbStatus, cluster resource.ID, alias string, upstreamAddresses []string) error {
+func (registry *Reconciler) ensureLB(cluster resource.ID, alias string, upstreamAddresses []string) error {
 	registry.logger.Log(registry.logLevel, "ensure LB", zap.String("cluster", cluster), zap.String("alias", alias), zap.Strings("upstreamAddresses", upstreamAddresses))
 
-	var existingUpstreamAddresses []string
+	hostPort := registry.hostPortForAlias(cluster, alias)
+	lbSts := registry.clusterToAliasToLBStatus[cluster][alias]
+
+	if lbSts == nil { // no LB yet, create and start it
+		tcpLB := &loadbalancer.TCP{
+			Logger: registry.logger.WithOptions(zap.IncreaseLevel(zapcore.ErrorLevel)),
```

Comment:
should create this one and re-use for all loadbalancers

Reply:
done